### ISSUE TITLE
Why not defining the current_resource_owner in doorkeeper.rb?

### ISSUE THREAD

**Issue by olivierobert** [ankitkalia1195/searchbot#6]
![ankit___volumes_workspace__candidates-tests_ankit__-_____app_controllers_api_v1_base_controller_rb__ankit_](https://user-images.githubusercontent.com/696529/39109805-b0176edc-46f8-11e8-8611-4114d80d9d9a.png)

Any specific reasons not to use `resource_owner_authenticator` in `config/initializers/doorkeeper.rb`?

![ankit___volumes_workspace__candidates-tests_ankit__-_____config_initializers_doorkeeper_rb__ankit_](https://user-images.githubusercontent.com/696529/39109792-a16e0df0-46f8-11e8-88ea-f9718c431326.png)

**Comment by ankitkalia1195** [comment]
Agreed it should be in doorkeeper.rb, will make the required changes.

**Comment by ankitkalia1195** [comment]
As i am using the doorkeeper in api_only mode and I have enabled only password grant type, i dont think this block is required.
